backend/api/guest_import/utils.py

```
@@ -0,0 +1,38 @@
+
+    This does NOT make the guest user available on the `request` object.
+    """
+    guest_token = request.COOKIES.get(GUEST_COOKIE_NAME)
+
+    if guest_token:
+        logger.debug("Guest session token: %s", guest_token)
+        try:
+            with transaction.atomic():
+                session = get_session(guest_token)
+                if not session:
+                    raise UserSession.DoesNotExist
```

Ruff (0.15.20)

[warning] Abstract `raise` to an inner function

(TRY301)